zend_abstract_interface/config/config_decode.c

```diff
@@ -0,0 +1,201 @@
+#include "config_decode.h"
+
+#include <assert.h>
+#include <ctype.h>
+#include <main/php.h>
+#include <stdbool.h>
+#include <string.h>
+#include <strings.h>
+
+#include "config.h"
+
+static bool zai_config_decode_bool(zai_string_view value, zval *decoded_value) {
```

Comment:
As per `zend_ini_parse_bool` at least on and yes are also valid values. (independently of whether we should validate or not.)

Reply:
Good point. We might want to adopt these two truthy values and their counterparts to be consistent with INI.